.github/workflows/dotnet-format-daily.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           action: "fix"
           #only-changed-files: true # only works for PRs
-          workspace: "Xamarin.Forms.sln"
+          workspace: "Microsoft.Maui.sln"
```

Comment:
does it need rebase? this change was in already

Reply:
rebased :)